pages/header_section.py

```diff
@@ -12,7 +12,7 @@ def click_sign_in_button(self):
         self.click(by_locator=self.__HEADER_SIGNIN_BUTTON)
 
     def check_if_nickname_is_presented(self):
-        return self.get_element(by_locator=self.__HEADER_NICKNAME_TEXT)
+        self.if_element_present(by_locator=self.__HEADER_NICKNAME_TEXT)
```

Comment:
`check...` functions have to return a boolean value that you can assert in `steps` implementation. so you need to type this way:
```python
    def check_if_nickname_is_presented(self) -> bool:
        return self.if_element_present(by_locator=self.__HEADER_NICKNAME_TEXT)
```